app/pages/lab-pages-editor/components/WorkflowSettingsPage.jsx

```diff
@@ -0,0 +1,255 @@
+          name="display_name"
+          defaultValue={workflow?.display_name || ''}
+          onBlur={doUpdate}
+        />
+      </label>
+
+      <div className="column-group col-1">
+        <fieldset>
+          <legend>Associated Subject Sets</legend>
+          <p>Choose the set of subjects you want to use for this workflow. Add subject sets in the Subject Sets tab.</p>
+          <p>TODO</p>
+        </fieldset>
+
+        <fieldset>
+          <legend>Associated Tutorial</legend>
+          <p>Choose the tutorials you want to use for this workflow. Create tutorials in the Tutorial tab.</p>
+          <p>TODO</p>
+        </fieldset>
+
+        <hr />
+
+        <fieldset>
+          <legend>Subject Retirement</legend>
+          <p id="subject-retirement-info">
+            Set how many people should classify each subject before it is
+            &quot;done.&quot; Once a subject has reached the retirement limit it
+            will no longer be shown to volunteers.
+          </p>
+          <div className="flex-row align-start spacing-bottom">
+            <select
+              aria-label="Retirement criteria"
+              className="flex-item"
+              defaultValue={workflow?.retirement?.criteria}
+              aria-describedby="subject-retirement-info"
+              name="retirement.criteria"
+              onChange={doUpdate}
+            >
+              <option value="classification_count">Classification count</option>
+              <option value="never_retire">Never retire</option>
+              {/* TODO: this is just a POC - never_retire should be removed, even though it's a valid option on the API. */}
+            </select>
+            <input
+              aria-label="Retirement count"
+              className="small-width"
+              defaultValue={workflow?.retirement?.options?.count}
+              data-updaterule="convert_to_number"
+              max="100"
+              min="1"
+              name="retirement.options.count"
+              onBlur={doUpdate}
+              placeholder="∞"
+              type="number"
+            />
+          </div>
+          <p className="small-info">
+            If you&apos;d like more complex retirement rules such as conditional
+            retirement using Caesar, please get in touch via the Contact Us
+            page.
+          </p>
+        </fieldset>
+      </div>
+
+      <div className="column-group col-2">
+        <fieldset>
+          <legend>Subject Viewer</legend>
+          <p id="subject-viewer-info">
+            Choose how to display your subjects.
+            Refer to the Subject Viewer section of the Glossary for more info.
+          </p>
+          <div className="flex-row align-start spacing-bottom">
+            <select
+              aria-label="Subject viewer"
+              className="flex-item"
+              data-updaterule="undefined_if_empty"
+              defaultValue={workflow?.configuration?.subject_viewer || ''}
```

Comment:
This should default to the viewer that's automatically set for you, but that requires a bunch of code that's buried in `@zooniverse/classifier` in the monorepo.